docusaurus/docs/core-concepts/config-store/overview.md

```diff
@@ -1,35 +1,56 @@
-# Configuration
-Ziti has the capability to define a configuration schema for an application and provide service configuration for those applications which need additional metadata. 
+# Configs
 
-## Why Centralized Configuration?
-One might ask why have this feature in Ziti when applications can store configuration data in local configuration files. While this approach works, centralized management makes deployments much easier. It can be difficult or impossible to update a file on a device out in the field, whereas updating the configuration in Ziti is easy, and running clients will quickly be notified of service changes.
+Configs are created and updated through the edge-management API and consumed by Edge SDKs through the edge-client API. Developers may define Config Types for storing arbitrary data in Ziti for use by their application. Configs and Config Types are Ziti entities. Each Config is an instance of a Config Type. A Config Type is defined by a JSON schema.
 
-The Ziti tunneler applicatons provide an example how configuration data can be used. 
+## Why Centralized Configuration?
```

Comment:
I'm not sure this section adds much value - it's basically pointing out that you don't need to copy your service configurations to every endpoint that uses them? I think that feature is table stakes, and mentioning it (here at least) distracts from the useful info about how to use service configurations.